pkg/templates/images/images.go

```diff
@@ -408,7 +410,9 @@ func optionalResources() map[Resource]map[string]string {
 		},
 
 		// operating-system-manager addon
-		OperatingSystemManager: {"*": "quay.io/kubermatic/operating-system-manager:v0.5.0"},
+		//  TODO: Update this to a semver tag before release.
+		// Ref: https://github.com/kubermatic/operating-system-manager/commit/45600227d69e6a105c34ba70ba641e5cd071887e
+		OperatingSystemManager: {"*": "quay.io/kubermatic/operating-system-manager:45600227d69e6a105c34ba70ba641e5cd071887e"},
```

Comment:
We should move this to `baseResources` because we're now deploying it by default.

Reply:
Good point, I'll update this 🦖